### ISSUE TITLE
Support Eslint (Google & AirBNB)

### ISSUE THREAD

**Issue by flowirtz** [FussyFox/lambdalint#1]
[Eslint](https://eslint.org/) is the goto js linting standard, some sort of support would be nice.
As it seems you'd prefer no configs therefore I'd suggest the Airbnb & Google styleguides as they are pretty big.

There are pretty well used implementations for both:
https://github.com/google/eslint-config-google
https://www.npmjs.com/package/eslint-config-airbnb

**Comment by codingjoe** [comment]
@FWirtz good idea. I haven't open sourced the JS linter code yet. I will do asap. With it I will be able to more quickly release new JS linters like ES lint.

BTW, we du support configurations. I will add documentation in the following days for each linter how to configure them. Of course we do not support additional dependencies which in the case of eslint is a restriction.

**Comment by codingjoe** [comment]
@amureki my JavaScript skills are limited, I can publish the existing code, but it needs some touches to be reusable like lintipy.

@FWirtz next time you organize an environment hackerthon, let me know in advance. I would have been happy to sponsor it, but my entire team is on vacation or conferences now.

**Comment by flowirtz** [comment]
@codingjoe sounds good, open sourcing that would be nice.

We had no idea, sorry. Thinking about doing another environment hack though, will keep u posted.

**Comment by codingjoe** [comment]
@FWirtz @amureki I posted the code form the standardjs function here:
https://github.com/LambdaLint/lambdalint-js

It is very old though and needs to be ported to support the new checks API. I would really welcome help here. My JavaScript is not very good.
